frontend/controller/router.js

```diff
@@ -170,12 +170,14 @@ const router = new Router({
       // beforeEnter: createEnterGuards(loginGuard, mailGuard)
       beforeEnter: createEnterGuards(inviteGuard)
     },
-    ...(process.env.NODE_ENV === 'development' && [{
-      path: '/error-testing',
-      name: 'ErrorTesting',
-      component: () => import('../views/pages/ErrorTesting.vue'),
-      meta: { title: L('Error Testing') }
-    }]),
+    ...(process.env.NODE_ENV === 'development'
+      ? [{
+          path: '/error-testing',
+          name: 'ErrorTesting',
+          component: () => import('../views/pages/ErrorTesting.vue'),
+          meta: { title: L('Error Testing') }
+        }]
+      : []),
```

Comment:
This is actually what I had before, and changed it to the `&&`  method. Curious, did you change it back because flow preferred this way?

Reply:
Yes, Flow complained about it, if I remember correctly (not 100% sure though)